docs/content/ingestion/data-formats.md

```diff
@@ -83,11 +83,14 @@ Since the CSV data cannot contain the column names (no header is allowed), these
     "columns" : ["timestamp","page","language","user","unpatrolled","newPage","robot","anonymous","namespace","continent","country","region","city","added","deleted","delta"],
     "dimensionsSpec" : {
       "dimensions" : ["page","language","user","unpatrolled","newPage","robot","anonymous","namespace","continent","country","region","city"]
-    }
+    },
+    "maxNumSkipHeadRows" : 5
   }
 ```
 
 The `columns` field must match the columns of your input data in the same order.
+Additionally, you can set the number of head rows to be skipped to `maxNumSkipHeadRows`. Note that this option is effective
```

Comment:
I would go further and say it's only effective for the non-Hadoop batch "index" task. Because it's also not useful for non-Hadoop realtime tasks; they wouldn't use FileIteratingFirehose.